docs/utilities/index.md

```diff
@@ -186,6 +186,8 @@ log.Infof("Log entry with format, %s", "additional log ~~message")
 
 log.Debugf("No changes detected in the API %s", *azAPI.Name)
 
+log.Trace("I got here in the code")
+
```

Comment:
what is this for?

Reply:
this is a file showing examples of how to use the log calls. There was no previous example of how to use Trace if a customer wanted to do such a thing in their agents.

Reply:
ok...